iree/turbine/kernel/wave/codegen.py

```diff
@@ -907,6 +909,40 @@ def handle_write(emitter: WaveEmitter, node: fx.Node):
             vector_d.scatter(kb_dest, start_indices, offsets_vec, mask, insert_vector)
 
 
+@handle_op(apply_expr)
+def handle_apply_expr(emitter: WaveEmitter, node: fx.Node):
+    try:
+        register, expr = node.args
+    except ValueError as e:
+        raise ValidationError("Malformed arguments") from e
+
+    APPLY_EXPR_ARG = index_symbol("$APPLY_EXPR_ARG")
+    expr = expr(APPLY_EXPR_ARG)
```

Comment:
@Hardcode84 could you explain the rationale here?
I was hoping this could compose expressions in a quite general manner but I must be using it improperly as the eval always returns False in my use cases.